lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -355,35 +355,19 @@ function dosomething_global_get_user_language($user = NULL) {
  * @return array
  *  All of the supported countries
  */
- function dosomething_global_get_countries() {
-   $lang_map = variable_get('dosomething_global_language_map', '');
-   $countries = array();
-   foreach ($lang_map as $lang) {
-     $country = $lang['country'];
-
-     // Prevents global english from slipping through
-     if ($country != NULL) {
-       array_push($countries, $country);
-     }
-   }
-   array_push($countries, 'global');
-   return $countries;
- }
+function dosomething_global_get_countries() {
+  $lang_map = variable_get('dosomething_global_language_map', '');
+  $countries = array();
+  foreach ($lang_map as $lang) {
+    $country = $lang['country'];
 
-/**
- * Returns the session's country code, or `global` if it's not
- * one of our whitelisted global countries.
- *
- * @return string
- *   country code, or 'global'
- */
-function dosomething_global_get_country() {
-  $country_code = dosomething_settings_get_geo_country_code();
-  if (in_array($country_code, dosomething_global_get_countries())) {
-    return $country_code;
+    // Prevents global english from slipping through
+    if ($country != NULL) {
+      array_push($countries, $country);
+    }
   }
-
-  return 'global';
+  array_push($countries, 'global');
+  return $countries;
```

Comment:
The diff is a little wonky, but `dosomething_global_get_countries` isn't removed. I'd recommend going commit-by-commit to make it easier to read.